src/Plugin/WSConnector/WSConnectorSimpleHTTPWithLangReplacement.php

```diff
@@ -53,6 +53,15 @@ public static function create(ContainerInterface $container, array $configuratio
     );
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getReplacements(array $options) {
+    $replacements = parent::getReplacements($options);
+    unset($replacements[array_search('LANGUAGE', $replacements)]);
```

Comment:
will LANGUAGE always be in the array of replacements? Just wondering what would happen if ever you end up with unset($replacements[FALSE])...

Reply:
In that context, LANGUAGE will always be there.